pom.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>de.thbingen.info2</groupId>
+    <artifactId>THBingen</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <kotlin.version>1.1.2</kotlin.version>
+        <main.class>HelloWorldKt</main.class>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.jetbrains.kotlin</groupId>
+            <artifactId>kotlin-stdlib</artifactId>
+            <version>${kotlin.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>com.univocity</groupId>
+            <artifactId>univocity-parsers</artifactId>
+            <version>2.4.1</version>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <sourceDirectory>${project.basedir}/src/main/kotlin</sourceDirectory>
+        <testSourceDirectory>${project.basedir}/src/test</testSourceDirectory>
```

Comment:
@zvenon: 

> Habe zusätzlich im Ordner 'test' einen Ordner namens 'kotlin' erstellt

Gut.

> und diesen als test source root markiert, jenes auch im Pfad in der pom abgeändert.

Wenn das Projekt als Maven-Projekt importiert ist, liest IntelliJ dinge wie Test Source Roots aus der POM aus und überschreibt damit andere Einstellungen - du solltest das also nur in der POM ändern.

> Jedoch funktioniert dann der Test nicht mehr.
> --> Maven: "only the Kotlin standard libary is allowed to use 'kotlin' package"

Das klingt so als würde der Ordner `kotlin` dann als Paket interpretiert. Das dürfen wir aber nicht benutzen, sondern nur Kotlin selber.

Es müsste gehen, wenn du nochmal den Ordner `kotlin` anlegst, `HelloWorldTest` dorthinein verschiebst (sicherstellen, dass die Datei nicht mit `package kotlin` beginnt - wenn doch die Zeile löschen) und dann den `testSourceDirectory` Pfad ebenfalls um `kotlin` erweiterst.